tensorflow/core/BUILD

```diff
@@ -1762,6 +1762,9 @@ tf_cc_tests(
         "graph/graph_test.cc",
         "graph/node_builder_test.cc",
         "graph/optimizer_cse_test.cc",
+# INTEL_MKL start
```

Comment:
I am not able to make that conditional through the select() function because there is a macro which iterates over all srcs and is not able to iterate over select():
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/tensorflow.bzl#L329
it throws error: "type 'select' is not iterable".
Is there any way to add conditional value beside select()?